multimodal/src/autogluon/multimodal/data/preprocess_dataframe.py

```diff
@@ -311,7 +311,7 @@ def _fit_y(self, y: pd.Series, X: Optional[pd.DataFrame] = None):
         elif self.label_type == NER_ANNOTATION:
             self._label_generator.fit(y, X[self._text_feature_names[0]])
         else:
-            raise NotImplementedError(f"Type of label column is not supported. Label column type={self._label_column}")
```

Comment:
IDENTIFIER is not support for the moment, right?

Reply:
Currently, Identifier is used for feature columns rather than the label column.

Reply:
I see